docs/pages/machine-id/guides/applications.mdx

```diff
@@ -0,0 +1,281 @@
+  https://grafana-example.example.teleport.sh/api/users
+```
+</ScopedBlock>
+<ScopedBlock scope={["oss","enterprise"]}>
+```code
+$ curl --user admin:admin \
+  --cert /opt/machine-id/tlscert \
+  --key /opt/machine-id/key \
+  https://grafana-example.teleport.example.com/api/users
+```
+</ScopedBlock>
+
+Note that in the example above, we include `--user` to provide a local username
+and password to our Grafana instance. This is not necessary if your application
+either requires no additional authentication, or if you've configured 
+[JWT authentication](../../application-access/jwt/introduction.mdx) for your
+app.
+
+No CA certificate needs to be specified so long as your Teleport Proxy is
+configured with a valid wildcard CA from Let's Encrypt or another public
+certificate authority.
+
+Note that if the certificates are invalid or otherwise misconfigured, clients
+will be redirected to the Teleport login page when attempting to access the app.
+
+<Admonition type="note" title="Proxy Note">
+While it's usually best to access apps through the public Teleport Proxy
+Service, it's also possible to open a local proxy to the app using
+`tbot proxy app ...` in situations where the Teleport Proxy Service's app endpoints are
+unavailable:
+
+<ScopedBlock scope={["cloud"]}>
+```code
+$ tbot proxy --destination-dir=/opt/machine-id --proxy=example.teleport.sh:443 app --port=1234 grafana-example
+```
+</ScopedBlock>
+<ScopedBlock scope={["oss","enterprise"]}>
+```code
+$ tbot proxy --destination-dir=/opt/machine-id --proxy=teleport.example.com:443 app --port=1234 grafana-example
+```
+</ScopedBlock>
+
+You may now connect to the app via a local proxy at `http://localhost:1234`:
+
+```code
+$ curl --user admin:admin http://localhost:1234/api/users
+```
+
+Note that any clients with access to `localhost` may connect to this proxy
+without additional authentication, so this should only be run in trusted
+environments.
+</Admonition>
+
+## Troubleshooting
+
+### Client application requires certificates with standard extensions
+
+If your client application requires TLS certificates with a specific file
+extension, you may also enable the `tls` config template in `tbot.yaml`:
+
+```yaml
+destinations:
+  - directory: /opt/machine-id
+    app: grafana-example
+
+    configs:
+      - tls
+```
+
+This will generate `tls.crt` and `tls.key` inside `/opt/machine-id`
+with identical content to the certificate files listed above.
+
+Be sure to re-run `tbot init ...` to configure permissions for the new file if
+config templates are changed.
+
+### Clients are redirected to the Teleport login page
+
+As with human users, scripted clients will be redirected to the Teleport login
+page when attempting to access an app through the Teleport Proxy Service without valid
+credentials.
+
+Ensure the bot's certificates have not expired and that the client application
+has been configured to use both the client certificate and key.
+
+## Next steps
+
+- Review the 
```

Comment:
We'll need to complete this

Reply:
Whoops, clearly my brain wasn't working. Will fix.